openbadge_analysis/visualization/audio_visualization.py

```diff
@@ -2,10 +2,9 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 import sys,os 
-src_dir = os.path.join(os.getcwd(), os.pardir, 'openbadge_analysis/preprocessing')
+src_dir = os.path.join(os.getcwd(), os.pardir, '../src')
```

Comment:
Not sure whether this would work inside a package. Isn't there a different way to import code from the same module?

Reply:
It works. I changed to this way because this style was used in "visualization/participation_interactive.py"

I tried it on a virtual environment before pushing to github.

Reply:
Sorry it took me so long to get to this. 
It's still not working when used in a package. This part is not what makes "participation_interactive" work. I'll take a look

(I should also cleanup participation_interactive at some point)

Reply:
ok, you just need to remove the "src_dir = .." and "sys.path.append(src_dir)" and add the following line:
from openbadge_analysis.preprocessing.audio import *

Note that it's actually a really bad habit to import * .